com.unity.render-pipelines.core/Runtime/Debugging/NvidiaDebugView.cs

```diff
@@ -0,0 +1,266 @@
+using System;
```

Comment:
Question: any reason this is in core vs HDRP given that DLSS is supported only in HDRP?